### PR TITLE
[Kaptain]: Managed and Attached clusters support

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-86797

## Description of changes being made
- Making sure support of Managed and Attached clusters is transparent to customers.
- Changed structure a bit to match existing docs

### Preview

http://docs-d2iq-com-pr-4340.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/install/index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ The infrastructure you select determines the specific requirements for a success
 
 ## Before you begin
 
-Kaptain is a DKP Catalog application. To use it, simply add it to your repository and deploy it on selected Workspaces. No downloads are necessary.
+Kaptain is a DKP Catalog application. To use it, add it to your repository and then deploy it on selected Workspaces. No downloads are necessary.
 
 You can deploy Kaptain to single and multi-clusters environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you run on one Management cluster only. In a multi-cluster environment (with an Enterprise license), you run Kaptain on one or several Managed and Attached clusters.
 
-Kaptain is deployed on a per-Workspace basis. If you are running a multi-cluster environment with several Workspaces, Kaptain is deployed to every cluster within a Workspace.
+You can deploy Kaptain on a per-Workspace basis. If you are running a multi-cluster environment with several Workspaces, Kaptain is deployed to every cluster within the selected Workspace. Workspaces can contain both Managed and Attached clusters, as a result, Kaptain deploys to both.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/install/index.md:30]: @chessahoekstra just want to be 100% that this is correct :)

**Review by juliangieseke — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/index.md (hunk 1 above)
Comment (chessahoekstra) [reply pages/dkp/kaptain/2.0.0/install/index.md:30]:
```suggestion
You can deploy Kaptain on a per-workspace basis. If you are running a multi-cluster environment, Kaptain is deployed to every cluster within the selected workspace.
```
this sounds good +concise to me - not sure if we need to specify that note about attached/managed clusters (though I did verify it it correct). but ultimately up to you on final touches!

Comment (chessahoekstra) [reply pages/dkp/kaptain/2.0.0/install/index.md:30]: @gvasquezvargas

**Review by chessahoekstra — COMMENTED** [review]
@alembiewski didn't you mention that knative is optional in 2.0.0?

**Comment by alembiewski** [comment]
@chessahoekstra @gvasquezvargas, yes, it is not required in the case of `RawDeployment` mode of KServe.
We can put the following note saying:
>  - KNative (optional, if KServe is configured to work in `RawDeployment` mode)
There are probably other pages mention KNative, we should probably mention that as well

File: pages/dkp/kaptain/2.0.0/install/index.md [hunk 2]
```diff
@@ -23,20 +23,20 @@ The infrastructure you select determines the specific requirements for a success
...
 
 ## Prerequisites
 
 -   A DKP cluster with the following Platform applications enabled:
 
     - Istio
-    - Knative
+    - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
 
 -   [`kubectl`][kubectl] on your installation machine
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/install/index.md:42]: (cant suggest changes for not changed line below this) shouldnt it be "Install overview"?

File: pages/dkp/kaptain/2.0.0/install/index.md [hunk 3]
```diff
@@ -23,20 +23,20 @@ The infrastructure you select determines the specific requirements for a success
 
 ## Before you begin
 
-Kaptain is a DKP Catalog application. To use it, simply add it to your repository and deploy it on selected Workspaces. No downloads are necessary.
+Kaptain is a DKP Catalog application. To use it, add it to your repository and then deploy it on selected workspaces. No downloads are necessary.
 
-You can deploy Kaptain to single and multi-clusters environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you run on one Management cluster only. In a multi-cluster environment (with an Enterprise license), you run Kaptain on one or several Managed and Attached clusters.
+You can deploy Kaptain to single and multi-clusters environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you deploy on one Management cluster only. In a multi-cluster environment (with an Enterprise license), you deploy Kaptain to one or several Managed or Attached clusters.
```
Comment (mmcelroy91) [inline pages/dkp/kaptain/2.0.0/install/index.md:28]:
```suggestion
You can deploy Kaptain to single and multi-clusters environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you deploy on one Management cluster only. In a multi-cluster environment (with an Enterprise license), Kaptain is deployed to either one or several Managed clusters or to Attached clusters.
```

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/index.md (hunk 2 above)
Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.0.0/install/index.md:42]: Yep, completely right!

**Pull request merged**